File: SO-API/SO.Domain/Ticket.cs
using SO.Domain.Enum;

namespace SO.Domain
{
    public class Ticket : BaseEntity
    {
        public string Protocol { get; set; }
        public string Subject { get; set; }
        public Complexity Complexity { get; set; }
        public Priority Priority { get; set; }
    }
}

File: SO-Front/src/app/demo/api/ticket.ts
export interface ticket {
    id?: number;
    protocol?: string;
    priority?: string;
    priorityId?: number;
    complexity?: string;
    complexityId?: number;
    subject?: string;
    createdBy?: string;
    changedBy?: string;
    createdDate?: Date;
    changedDate?: Date;
}
